typings/index.d.ts

```diff
@@ -0,0 +1,477 @@
+
+    // Worksheets
+    readonly sheetsById: Record<string, GoogleSpreadsheetWorksheet>
+    readonly sheetsByIndex: GoogleSpreadsheetWorksheet[]
+    readonly sheetCount: number
+
+    constructor(spreadsheetId: string)
+
+    // Authentication
+    useServiceAccountAuth(creds: {
+      client_email: string
+      private_key: string
+    }): Promise<void>
+    useApiKey(key: string): Promise<void>
+
+    // Basic info
+    loadInfo(): Promise<void>
+    updateProperties(props: GoogleSpreadsheetBase): Promise<void>
+    resetLocalCache(): void
+
+    // Managing Sheets
+    addSheet(props?: {
+      sheetId?: number
+      headerValues?: string[]
+      props?: GoogleSpreadsheetWorksheetBase
+    }): Promise<GoogleSpreadsheetWorksheet>
+    deleteSheet(sheetId: string): Promise<void>
+
+    // Named Ranges
+    addNamedRange(name: string, range: string | GridRange, rangeId?: string): Promise<any>
+    deleteNamedRange(rangeId: string): Promise<any>
+
+    // "Private" methods (not documented)
+    private renewJwtAuth(): Promise<void>
+    private _setAxiosRequestAuth<T>(config: T): Promise<T>
+    private _handleAxiosResponse<T>(response: T): Promise<T>
+    private _handleAxiosErrors(error): Promise<void>
+    private _makeSingleUpdateRequest(requestType, requestParams): Promise<any>
+    private _makeBatchUpdateRequest(requests, responseRanges): Promise<void>
+    private _ensureInfoLoaded(): void
+    private _updateRawProperties(newProperties): void
+    private _updateOrCreateSheet({ properties, data }): void
+    private _getProp(param)
+    private _setProp(param, newVal): never
+    private loadCells(filters): Promise<void>
+  }
+
+  interface GoogleSpreadsheetWorksheetBase {
+    title?: string
+    index?: number
+    gridProperties?: GridProperties
+    hidden?: boolean
+    tabColor?: Color
+    rightToLeft?: boolean
+  }
+  export class GoogleSpreadsheetWorksheet implements GoogleSpreadsheetWorksheetBase {
+    // Basic sheet properties
+    readonly sheetId: string
+    readonly sheetType: SheetType
+    readonly title: string
+    readonly index: number
+    readonly gridProperties: GridProperties
+    readonly hidden: boolean
+    readonly tabColor: Color
+    readonly rightToLeft: boolean
+
+    // Sheet Dimensions & Stats
+    readonly rowCount: number
+    readonly columnCount: number
+    readonly cellStats: {
+      total: number
+      nonEmpty: number
+      loaded: number
+    }
+
+    constructor(parentSpreadsheet: GoogleSpreadsheet, { properties, data })
+
+    // Working With Rows
+    loadHeaderRow(): Promise<void>
+    setHeaderRow(headerValues: string[]): Promise<void>
+    addRow(values: object): Promise<GoogleSpreadsheetRow>
+    getRows(options?: {
+      offset?: number
+      limit?: number
+    }): Promise<GoogleSpreadsheetRow>
+
+    // Working With Cells
+    loadCells(filters?: any): Promise<any>
+    getCells(rowIndex: number, columnIndex: number): GoogleSpreadsheetCell
```

Comment:
```suggestion
    getCell(rowIndex: number, columnIndex: number): GoogleSpreadsheetCell
```

Looks like a typo compared to [doc](https://theoephraim.github.io/node-google-spreadsheet/#/classes/google-spreadsheet-worksheet?id=fn-getcell)